### PR TITLE
Testcase/lvv-t2213 - Look-up Table Application from MTMount Elevation and Azimuth Changes

### PR DESCRIPTION
**Pull request by b1quint**
Updated the notebooks to run and analyze the data for LVV-T2213. The execution notebook seems to be in a final state. The data-analysis notebook might need some improvements in the future but, for now, the current content should be sufficient.

I ran this notebook on TTS and I do have some questions regarding the compensated position of the hexapods. It is not clear to me yet when they should be published.

### CODE REVIEW

File: notebooks/proj_sys_eng/sitcom_integration/l3_system_integ/LVV-T2213-LUT_from_elevation_changes.ipynb [hunk 1]
```diff
@@ -391,37 +257,24 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "# We are running this command because we do not want to track\n",
-    "await mtcs.rem.mtmount.cmd_moveToTarget.set_start(azimuth=0, elevation=90)"
+    "await mtcs.point_azel(az=0, el=90)"
```
Comment (isotuela) [inline notebooks/proj_sys_eng/sitcom_integration/l3_system_integ/LVV-T2213-LUT_from_elevation_changes.ipynb]: Move in small steps to the zenith, similar to the  `safe` = False` option below.

File: notebooks/proj_sys_eng/sitcom_integration/l3_system_integ/LVV-T2213-LUT_from_elevation_changes.ipynb [hunk 2]
```diff
@@ -1125,26 +653,27 @@
...
-    "Check the camera hexapod LUT compensations match what we expect from the implemented LUT"
+    "if safe:\n",
+    "    print(datetime.now())\n",
+    "    script.log.info(f\"MOVING - {test_case} {test_exec} Moving to az/el = 90./84.\")\n",
+    "    await mtcs.point_azel(az=90, el=84)\n",
+    "    await asyncio.sleep(39.)\n",
+    "    \n",
+    "    print(datetime.now())\n",
+    "    script.log.info(f\"MOVING - {test_case} {test_exec} Moving to az/el = 180./85.\")\n",
+    "    await mtcs.point_azel(az=180, el=85)\n",
```
Comment (isotuela) [inline notebooks/proj_sys_eng/sitcom_integration/l3_system_integ/LVV-T2213-LUT_from_elevation_changes.ipynb:659]: Why changing the elevation to 85 degrees, as well?

File: notebooks/proj_sys_eng/sitcom_integration/l3_system_integ/LVV-T2213-LUT_from_elevation_changes.ipynb [hunk 3]
```diff
@@ -1201,10 +710,11 @@
    "source": [
     "# Put the telescope back to the original position\n",
     "# Specially if running at TTS\n",
-    "target = mtcs.radec_from_azel(az=0, el=80)\n",
+    "if safe:\n",
+    "    target = mtcs.radec_from_azel(az=0, el=80)\n",
```
Comment (isotuela) [inline notebooks/proj_sys_eng/sitcom_integration/l3_system_integ/LVV-T2213-LUT_from_elevation_changes.ipynb:709]: Please add the `not Safe` option

File: notebooks/proj_sys_eng/sitcom_integration/l3_system_integ/LVV-T2241-MTAOS_accumulation.ipynb [hunk 4]
```diff
@@ -0,0 +1,897 @@
...
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "7eb81386",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "await mtcs.slew_object(target, rot_type=RotType.PhysicalSky, rot=1.9)"
```
Comment (isotuela) [inline notebooks/proj_sys_eng/sitcom_integration/l3_system_integ/LVV-T2241-MTAOS_accumulation.ipynb:549]: Perhaps add here safe = False to move mount in small steps?

File: notebooks/proj_sys_eng/sitcom_integration/l3_system_integ/LVV-T2241-MTAOS_accumulation.ipynb [hunk 5]
```diff
@@ -0,0 +1,897 @@
...
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "951661e0-974c-4dec-8c68-03b7407f01ed",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "await mtcs.point_azel(az=0, el=80)\n",
```
Comment (isotuela) [inline notebooks/proj_sys_eng/sitcom_integration/l3_system_integ/LVV-T2241-MTAOS_accumulation.ipynb:784]: Same here. Perhaps add here safe = False to move mount in small steps?

**Review by isotuela — COMMENTED** [review]
It looks good. I just noted some instances in the execution notebooks for LVV-T2213 and LVV-T2241 in which the `safe=False` option to move the mount in small steps could be added for added safety.

File: notebooks/proj_sys_eng/sitcom_integration/l3_system_integ/LVV-T2213-LUT_from_elevation_changes.ipynb (hunk 3 above)
Comment (b1quint) [reply notebooks/proj_sys_eng/sitcom_integration/l3_system_integ/LVV-T2213-LUT_from_elevation_changes.ipynb:709]: Done.

File: notebooks/proj_sys_eng/sitcom_integration/l3_system_integ/LVV-T2213-LUT_from_elevation_changes.ipynb (hunk 1 above)
Comment (b1quint) [reply notebooks/proj_sys_eng/sitcom_integration/l3_system_integ/LVV-T2213-LUT_from_elevation_changes.ipynb]: Done.

File: notebooks/proj_sys_eng/sitcom_integration/l3_system_integ/LVV-T2213-LUT_from_elevation_changes.ipynb (hunk 2 above)
Comment (b1quint) [reply notebooks/proj_sys_eng/sitcom_integration/l3_system_integ/LVV-T2213-LUT_from_elevation_changes.ipynb:659]: I don't know. This comes from the test case.

File: notebooks/proj_sys_eng/sitcom_integration/l3_system_integ/LVV-T2241-MTAOS_accumulation.ipynb (hunk 4 above)
Comment (b1quint) [reply notebooks/proj_sys_eng/sitcom_integration/l3_system_integ/LVV-T2241-MTAOS_accumulation.ipynb:549]: This is a stray commit that comes from another test case. This change should not be here.

File: notebooks/proj_sys_eng/sitcom_integration/l3_system_integ/LVV-T2241-MTAOS_accumulation.ipynb (hunk 5 above)
Comment (b1quint) [reply notebooks/proj_sys_eng/sitcom_integration/l3_system_integ/LVV-T2241-MTAOS_accumulation.ipynb:784]: Same here. This is a stray commit. It should be updated on another branch/PR.

**Review by isotuela — APPROVED** [review]
Thank you so much. I like the added Safer option.

**Pull request merged**